core/types/reqeusts_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014 The go-ethereum Authors
```

Comment:
No need for the go-ethereum copyright (unless it was taken from geth's codebase)